packages/react-instantsearch/src/ui/CurrentRefinements.tsx

```diff
@@ -97,7 +96,7 @@ export function CurrentRefinements({
             <span
               className={cx('ais-CurrentRefinements-label', classNames.label)}
             >
-              {item.label}:
+              {capitalize(item.label)}:
```

Comment:
I pondered the idea of doing the reverse, but both IS.js and Vue have capitalization for the label. I don't think this is a breaking change.

Reply:
I think this is ok, but would also be ok with leaving the difference and later in a major converging on lowercase